src/Utils.ts

```diff
@@ -1048,13 +1060,16 @@ class DummyInputManagerCore {
 
         // Some elements allow only specific types of direct descendants and we need to
         // put our dummy inputs inside or outside of the element accordingly.
-        const tagName = element.get()?.tagName;
+        const forcedDummyPosition = sys?.dummyInputsOutside;
```

Comment:
Something like this would be cleaner

```ts
switch(tagName) {
  case 'UL':
  case 'OL':
  case 'Table':
    this._isOutside = true;
    break;
  default:
    this._isOutside = false;
}

this._isOutside = forcedDummyPosition ?? this._isOutside;

```

Reply:
I will adjust it, but it is much less bundle size friendly :).

Reply:
Actually, the condition is more complex than that and rewriting it to switch/cases makes it more monstrous...